lib/model.ts

```
@@ -29,6 +29,17 @@ export class Model<T extends Record<string, any>> { // Ensure T is an object
         return true;
     }
 
+    validatePartial(data: Partial<T>): boolean {
+        const partialSchema: any = { ...this.schema };
+        delete partialSchema.required;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)